### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Korusuke**
Are there anymore things that you would like to be there in this ReadMe?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
+
 # Universal Packaging Tool: MacPorts backend
 This is a [MacPorts](https://www.macports.org) backend for [upt](https://pypi.python.org/pypi/upt).
 
-#### Steps to use this tool out:
+## Installation
+
+### Using MacPorts
+
+> port install py-upt-macports
```
Comment (Steap) [inline README.md]: Does this install all frontends?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,25 @@
+
```
Comment (Steap) [inline README.md]: Why the blank line?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+**Note**: upt can be installed from [pypi](https://pypi.org/project/upt/) or [framagit](https://framagit.org/upt/upt)
+```
+git clone https://github.com/macports/upt-macports.git
+cd upt-macports/
+python3 setup.py install
+```
+
+**Note**: To use recursive and update feature MacPorts should be installed. For Installing MacPorts refer this [guide](https://www.macports.org/install.php). To install MacPorts in docker refer [this](https://github.com/Korusuke/MacPorts-Docker).
+
```
Comment (Steap) [inline README.md:31]: I'd say "to use *the* recursive and update features" and "refer to" (twice).

File: README.md [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+## Usage
+
+Refer [upt guide](https://framagit.org/upt/upt/blob/master/README.md).
```
Comment (Steap) [inline README.md]: Refer *to*.

**Comment by Steap** [comment]
Can you add "Closes: #54" in the commit log? It should automatically close the corresponding issue once the patch is merged?

File: README.md (hunk 1 above)
Comment (Korusuke) [reply README.md]: yep, now it does

File: README.md [hunk 5]
```diff
@@ -1,8 +1,23 @@
...
 
-#### Steps to use this tool out:
+## Installation
 
-- Clone this repository and run ```python3 setup.py install``` to install macports backend
-- Run ```$ upt package -f pypi -b macports package_name```
-- Portfile output will be printed via stdout
+### Using MacPorts
+
+> port install py-upt
```
Comment (reneeotten) [inline README.md]: this should be: ```sudo port install upt```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,23 @@
 # Universal Packaging Tool: MacPorts backend
 This is a [MacPorts](https://www.macports.org) backend for [upt](https://pypi.python.org/pypi/upt).
```
Comment (reneeotten) [inline README.md:2]: here you should add that work on this was done during the GSoC 2019 period, etcetera....

**Review by reneeotten — CHANGES_REQUESTED** [review]
a few more comments

**Comment by reneeotten** [comment]
@Korusuke @Steap I made a few modifications to the text; please let me know what you think. If there are no further comments I intend to merge this on Saturday morning (EST time).

File: README.md [hunk 7]
```diff
@@ -1,8 +1,33 @@
...
 
-#### Steps to use this tool out:
+The prototype was written by Mojca Miklavec (@mojca) and further developed as part of
+[Google Summer of Code 2019](https://summerofcode.withgoogle.com/archive/2019) by
+Karan Seth (@Korusuke), mentored by Cyril Roelandt (@Steap) and Renee Otten (@reneeotten).
```
Comment (Korusuke) [inline README.md]: Karan Sheth

File: README.md [hunk 8]
```diff
@@ -1,8 +1,33 @@
...
+
+### Install from source
+The source can be obtained from [PyPI](https://pypi.org/project/upt/) or [framagit](https://framagit.org/upt/upt); several
+frontends are also available from these websites.
+
+To install the MacPorts backend, first clone the repository and follow the typical installation steps:
+```
+git clone https://github.com/macports/upt-macports.git
+cd upt-macports/
+python setup.py install
```
Comment (Korusuke) [inline README.md:25]: Should we write `python3` instead as for systems with both versions installed `python` would result in `python2`, at least this is the behavior for my ubuntu installation.

File: README.md (hunk 7 above)
Comment (reneeotten) [reply README.md]: sorry about that... fixed

**Review by reneeotten — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (reneeotten) [reply README.md:25]: okay, I added a sentence to clarify what ```python``` should be

**Pull request merged**